src/Core/Services/StateService.cs

```diff
@@ -1359,6 +1359,39 @@ public void SetConfigs(ConfigResponse value)
             _storageMediatorService.Save(Constants.ConfigsKey, value);
         }
 
+        public async Task<Region?> GetActiveUserRegionAsync(string userId = null)
+        {
+            return (await GetAccountAsync(
+                ReconcileOptions(new StorageOptions { UserId = userId }, await GetDefaultStorageOptionsAsync())
+            ))?.Settings?.Region;
+        }
```

Comment:
🎨 Naming doesn't reflect the method given that it says **active user** and given that the `userId` is being passed, it could be a different user than the active one. If the intent is to get the active one, then remove the parameter and use:
```suggestion
        public async Task<Region?> GetActiveUserRegionAsync()
        {
            return await GetActiveUserCustomDataAsync(a => a?.Settings?.Region);
        }
```
Otherwise, rename to `GetRegionAsync(string userId = null)`